package.json

```diff
@@ -64,7 +64,7 @@
     },
     {
       "name": "Calyhre",
-      "email": "charley.david@wopata.com"
+      "email": "calyhre@gmail.com"
```

Comment:
The contributors actually get auto-updated with a script before release, but this is fine.